src/utils/slpReader.ts

```diff
@@ -211,12 +218,13 @@ export function iterateEvents(
 
   // Generate read buffers for each
   const commandPayloadBuffers = mapValues(slpFile.messageSizes, (size) => new Uint8Array(size + 1));
+  let splitMessageBuffer = new Uint8Array(0);
 
   const commandByteBuffer = new Uint8Array(1);
   while (readPosition < stopReadingAt) {
     readRef(ref, commandByteBuffer, 0, 1, readPosition);
-    const commandByte = commandByteBuffer[0] as number;
-    const buffer = commandPayloadBuffers[commandByte];
+    let commandByte = (commandByteBuffer[0] as number) ?? 0;
```

Comment:
What are the implications of this default commandByte `0`? Why would the command byte be undefined?

Reply:
Under any normal circumstance it wouldn't be. It's just a fallback value to appease the type system. Actually with this the `as number` could be removed.